pages/common/paci.md

```diff
@@ -0,0 +1,23 @@
+# paci
+
+> Your friendly, lightweight and flexible package manager. Written in Python.
```

Comment:
We aim to be objective and formal in our descriptions. Let's remove "Your friendly".

How about - "A lightweight and flexible package manager for Python applications" ?

Reply:
I understand. I would propose "A lightweight and flexible package manager for bash scripts."

Because the tool itself is written in Python but the packages use bash.